core/domain/skill_domain.py

```diff
@@ -378,3 +378,31 @@ def to_dict(self):
             'skill_model_created_on': self.skill_model_created_on,
             'skill_model_last_updated': self.skill_model_last_updated
         }
+
+
+class SkillsMastery(object):
+    """Domain object for an Oppia Skills Mastery."""
+
+    def __init__(self, user_id, skill_id, degree_of_mastery):
+        """Constructs a SkillMastery domain object.
+
+        Args:
+            skill_id: str. The unique id of the skill.
+            user_id: str. The user id of the logged in user.
+            degree_of_mastery: float. The language code of the skill.
```

Comment:
Can the docstrings be arranged in the same order as the args?
Also, wrong docstring?